src/main/java/com/hashedin/redmask/common/DataMasking.java

```diff
@@ -142,9 +161,46 @@ public void buildQueryAndAppend(MaskingRule rule, FileWriter writer,
       throw new RedmaskRuntimeException(
           "Exception while appending to file writer", e);
     }
+  }
 
+  /**
+   * This function create the table for housing the masked data when the mode is static
+   *
+   * @param rule           The masking rule given for a table.
+   * @param writer         FileWriter object. All the queries are appended to this writer.
+   * @param config         MaskConfiguration object
+   * @param querySubstring List of  masked column queries
+   */
+  void createTableQuery(MaskingRule rule, FileWriter writer, MaskConfiguration config,
+                        List<String> querySubstring) {
+    // Create view query for Dynamic masking.
+    log.info("Creating the query in order to create the intended masked view.");
+    String queryString = String.join(",", querySubstring);
+    StringBuilder sb = new StringBuilder();
+    if (config.getDbType().toString().equalsIgnoreCase(DataBaseType.SNOWFLAKE.toString())) {
+      sb.append("public.");
+    }
+    sb.append("mask_").append(rule.getTable());
+
+    String createTableQuery = "CREATE TABLE " + sb.toString() + " AS SELECT "
+        + queryString + " FROM ";
+    if (config.getDbType().toString().equalsIgnoreCase(DataBaseType.SNOWFLAKE.toString())) {
+      createTableQuery = createTableQuery + "public." + rule.getTable() + ";";
+    } else {
+      createTableQuery = createTableQuery + rule.getTable() + ";";
+    }
+    try {
+      //Dropping existing table if exist
+      writer.append("\nDROP TABLE IF EXISTS " + sb.toString() + ";");
```

Comment:
why `DROP TABLE IF EXISTS` and which table you are dropping.

Reply:
I am dropping the mask<table_name> table as not all database will support the create or replace table as select command.